google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java

```diff
@@ -107,9 +106,6 @@ public Void apply(WriteLogEntriesResponse input) {
 
   private static <I, O> Future<O> transform(Future<I> future,
       Function<? super I, ? extends O> function) {
-    if (future instanceof ListenableFuture) {
-      return Futures.transform((ListenableFuture<I>) future, function);
-    }
```

Comment:
Since we are switching to using our own `RpcFuture` which has `addCallback` functionality, this workaround will no longer be necessary.